### DISCUSSION TITLE
Latest relaease "<c-/>", lazyterm toggle not working?

### DISCUSSION THREAD

**Discussion by ibeex** [LazyVim/LazyVim#880] [Q&A] [answered]
hi,
in last release https://github.com/LazyVim/LazyVim/commit/7a36e2989c3d62e8dbaf4036f5c4551929c565a5 
there is `CTRL /` option to toggle terminal but I can't get this working? Is there anything else needed for it to work?

**Comment by folke** [comment]
Check your terminal settings. Especially when you're using iterm2

**Reply by ibeex** [reply]
I am using tmux in alacritty on linux? which settings should I check? <leader>ft[T] works.

**Reply by folke** [reply]
I have no idea. I'm using kitty.

**Comment by jyuan0** [comment, marked as answer]
You can try adding a map for `<c-_>`.

https://vi.stackexchange.com/questions/26611/is-it-possible-to-map-control-forward-slash-with-vim

**Reply by dpetka2001** [reply]
Yep that did the trick :smile:

**Comment by ibeex** [comment]
great adding  that to keymaps.lua fixed it. Thx @jyuan0 
```
local Util = require("lazyvim.util")
local lazyterm = function() Util.float_term(nil, { cwd = Util.get_root() }) end
vim.keymap.set("n", "<c-_>", lazyterm, { desc = "Terminal (root dir)" })
vim.keymap.set("t", '<C-_>', "<cmd>close<cr>", {desc = "Hide Terminal"})
```

**Reply by jyuan0** [reply]
If you'd rather not duplicate the code, a recursive map works for me:
```
vim.keymap.set("n", "<c-_>", "<c-/>", { remap = true, desc = "which_key_ignore" })
vim.keymap.set("t", "<c-_>", "<c-/>", { remap = true, desc = "which_key_ignore" })
```

Change `which_key_ignore` if you want to see the label.

**Reply by dpetka2001** [reply]
Neat trick. Thank you for sharing.

**Reply by dvic** [reply]
I can also confirm that the mapping works with kitty but does not with iterm2. According to [this](https://apple.stackexchange.com/a/24282) `ctrl+/` is not part of the standard control codes? Maybe we can add the extra remapping "<c-_>" as well? (this also makes `ctrl+/` work for me on iterm2)

**Reply by dpetka2001** [reply]
You can just add it to your own keymaps spec. That's what we all did.

**Reply by dvic** [reply]
> You can just add it to your own keymaps spec. That's what we all did.

Yes of course, I also did this, but I meant for future users encountering this problem :).

**Reply by folke** [reply]
Good idea. Just added this.

**Reply by jyuan0** [reply]
Odd, I just updated and if I remove my maps, it doesn't work :shrug:

**Reply by dvic** [reply]
> Odd, I just updated and if I remove my maps, it doesn't work 🤷

Hmm that's weird, for the record: for me it does work after removing my custom mappings.

**Reply by ibeex** [reply]
also works for me with new settings

**Reply by dpetka2001** [reply]
It does work with the latest update. Not with the previous one which @jyuan0 must have encountered. Just tried it before the update and wasn't working for me neither in tmux, but worked as expected after the latest update.

**Reply by folke** [reply]
Yep, my bad. Properly fixed now 🙂

**Reply by dpetka2001** [reply]
If you don't mind me asking why didn't it work with the recursive mapping, though it worked in our personal configs, and you had to revert to specifying the `lazyterm` command? Does it have to do to being specified in the lazyvim config instead of the user's personal config?

**Reply by folke** [reply]
It did work, but in order not to duplicate keymaps for vscode, I added a bunch of `remap=true`, but those remaps are only enabled when in vscode.
I need to properly fix that at some point :)

**Comment by directormac** [comment]
If you guys are using the new edgy extra plugin the <C-/> works like a charm, thank you @folke !
